cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -2558,13 +2582,94 @@ def test_development_stage_ontology_term_id_zebrafish__invalid(
             "'development_stage_ontology_term_id' MUST be the most accurate descendant of 'ZFS:0100000' and it "
             "MUST NOT be 'ZFS:0000000' for Unknown. The str 'unknown' is acceptable."
         )
-        validator = validator_with_adata
+        validator = validator_with_zebrafish_adata
         obs = validator.adata.obs
-        obs.loc[obs.index[0], "organism_ontology_term_id"] = "NCBITaxon:7955"
         obs.loc[obs.index[0], "development_stage_ontology_term_id"] = development_stage_ontology_term_id
-        obs.loc[
-            obs.index[0],
-            "self_reported_ethnicity_ontology_term_id",
-        ] = "na"
         validator.validate_adata()
         assert validator.errors == [error + " " + zebrafish_error_message_suffix]
+
+    @pytest.mark.parametrize(
+        "organism_cell_type_ontology_term_id",
+        ["ZFA:0000003", "unknown"],
```

Comment:
can you also add a non-zebrafish organism + `na` as a valid case?

Reply:
that case is covered in test_valid_anndata, since the example anndata already has non-zebrafish organisms + na